src/v/redpanda/admin/api-doc/status.json

```diff
@@ -11,4 +11,35 @@
         }
       }
     }
+  },
+  "/v1/status/started": {
+    "get": {
+      "summary": "redpanda started",
+      "operationId": "started",
+      "responses": {
+        "200": {
+          "description": "Redpanda is actively participating in the cluster",
+          "type": "object",
+          "properties": {
+            "message": {
+              "type": "string"
+            },
+            "code": {
+              "type": "integer"
+            }
+          }
+        },
+        "503": {
+          "description": "Redpanda is not actively participating in the cluster",
```

Comment:
I'm unsure how a user will interpret this text if they're looking, they might think action is required?  Maybe we should say something like "Redpanda node is starting up" to be clear that this node is not stuck.